guides/common/modules/proc_creating-custom-ssl-certificates-for-smart-proxy-server.adoc

```diff
@@ -57,7 +51,22 @@ Ensure to change this when running the command on each {SmartProxyServer} that y
 You can also set a wildcard value `*`.
 If you set a wildcard value, you must add the `-t {certs-proxy-context}` option when you use the `katello-certs-check` command.
 <2> Under `[alt_names]`, include the FQDN of the load balancer as `DNS.1` and the FQDN of {SmartProxyServer} as `DNS.2`.
-
+Optional: If you want to add Distinguished Name (DN) details to the CSR, add the following information to the `[ req_distinguished_name ]` section:
```

Comment:
we don't have this section in https://docs.theforeman.org/nightly/Installing_Server/index-katello.html#Configuring_Server_with_a_Custom_SSL_Certificate_foreman, should we add it there too?

Reply:
I have filed a PR for this https://github.com/theforeman/foreman-documentation/pull/2828/files
I'll make changes in this file (including dropping the x509 sections) once this PR is merged.

Reply:
@evgeni done!